lib/cisco_node_utils/cmd_ref/vrf.yaml

```diff
@@ -18,9 +17,37 @@ description:
 destroy:
   config_set: "no vrf context <vrf>"
 
+feature_vn_segment_vlan_based:
+  _exclude:
+    - /N7K/  # Support for multi-tenancy full only
```

Comment:
I have no feature vn_segment-vlan-based on N9K and run the test. I got the feature vn-segment-vlan-based, so vn_segment config_get_token should be called.